src/test/java/com/google/sps/servlets/GetNearbyVendorsTest.java

```diff
@@ -0,0 +1,230 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.only;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.COMMONS;
+import com.google.sps.servlets.GetNearbyVendorsServlet;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class GetNearbyVendorsTest {
+  
+  // Request parameters names
+  private static final String PARAM_DELIVERY = "hasDelivery";
+  private static final String PARAM_OPEN = "onlyOpenNow";
+  private static final String PARAM_TIME = "currentTime";
+  private static final String PARAM_DISTANCE = "distance";
+  private static final String PARAM_LATITUDE = "lat";
+  private static final String PARAM_LONGITUDE = "lng";
+
+  // Delivery and open values
+  private static final String TRUE = "true";
```

Comment:
Why is it necessary to declare this variable?

Reply:
My reasoning was to avoid typos, since `"true"` is a string not the boolean value. 😅